base/ltfssaxes.dtx

```diff
@@ -1299,38 +1368,176 @@
 \DeclareFontShapeChangeRule {scsw}{sw}  {scsw}  {}
 \DeclareFontShapeChangeRule {scsw}{sc}  {scsw}  {}
 \DeclareFontShapeChangeRule {scsw}{ulc} {sw}    {}
-\DeclareFontShapeChangeRule {scsw}{up} {sc}     {}
+\DeclareFontShapeChangeRule {scsw}{up}  {sc}    {}
+\DeclareFontShapeChangeRule {scsw}{ssc} {sscsw} {scsw}
 %    \end{macrocode}
 %
 %    \begin{macrocode}
 %\DeclareFontShapeChangeRule {sw}{n}   {n}     {}
 %\DeclareFontShapeChangeRule {sw}{it}  {it}    {}
 %\DeclareFontShapeChangeRule {sw}{sl}  {sl}    {}
 %\DeclareFontShapeChangeRule {sw}{sw}  {sw}    {}
-\DeclareFontShapeChangeRule {sw}{sc}  {scsw}  {}
+\DeclareFontShapeChangeRule {sw}{sc}  {scsw}  {scit}
 \DeclareFontShapeChangeRule {sw}{ulc} {sw}    {}
-\DeclareFontShapeChangeRule {sw}{up} {n}      {}
+\DeclareFontShapeChangeRule {sw}{up}  {n}     {}
+\DeclareFontShapeChangeRule {sw}{ssc} {sscsw} {scsw}
+%    \end{macrocode}
+%    
+%    \begin{macrocode}
+%\DeclareFontShapeChangeRule {ssc}{n}   {n}     {}
+\DeclareFontShapeChangeRule {ssc}{it}  {sscit}  {sscsl}
+\DeclareFontShapeChangeRule {ssc}{sl}  {sscsl}  {sscit}
+\DeclareFontShapeChangeRule {ssc}{sw}  {sscsw}  {sscit}
+\DeclareFontShapeChangeRule {ssc}{sc}  {sc}     {}
+%\DeclareFontShapeChangeRule {ssc}{ssc}  {ssc}    {}
+\DeclareFontShapeChangeRule {ssc}{ulc} {n}      {}
+%    \end{macrocode}
+%    We implement the same logic as for \texttt{sc}, see above. The
+%    \texttt{ssc} shape doesn 't to care for 30 years of history, but
```

Comment:
Something wrong here! Maybe it should be something like this:

. . . shape doesn't have/need to care about 30 years . . .